apps/coscrad-frontend/src/coscrad-theme-provider.tsx

```diff
@@ -10,13 +10,15 @@ interface CoscradThemeProviderProps {
 export const CoscradThemeProvider = ({ children }: CoscradThemeProviderProps): JSX.Element => {
     const _configurableContent = useContext(ConfigurableContentContext);
 
-    const coscradDefaultTheme = createTheme({
+    let coscradDefaultTheme = createTheme({
```

Comment:
Why did you change this to `let`? There is never a reason to reassign the reference and we should probably even be careful mutating this.

Reply:
I was playing around with their responsive font sizes.  This is the model they have for implementing MUI responsive font sizes with `responsiveFontSizes()`:

`coscradDefaultTheme = responsiveFontSizes(coscradDefaultTheme);`

<img width="703" alt="image" src="https://user-images.githubusercontent.com/63014783/224086266-822cd89b-07e3-4094-ad49-bfe495027e42.png">

https://mui.com/material-ui/customization/typography/#responsive-font-sizes

Reply:
Refactored back to const by passing the `createTheme` call into the declaration.